src/components/landing/rework/landing-subjects-new.tsx

```diff
@@ -0,0 +1,317 @@
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { lighten } from 'polished'
+import styled, { keyframes } from 'styled-components'
+
+import AbcSVG from '@/assets-webkit/img/landing/subjects-abc.svg'
+import BiologySVG from '@/assets-webkit/img/landing/subjects-biology.svg'
+import BlankSVG from '@/assets-webkit/img/landing/subjects-blank.svg'
+import ChemistrySVG from '@/assets-webkit/img/landing/subjects-chemistry.svg'
+import InformaticsSVG from '@/assets-webkit/img/landing/subjects-informatics.svg'
+import MathSVG from '@/assets-webkit/img/landing/subjects-math.svg'
+import NewSVG from '@/assets-webkit/img/landing/subjects-new.svg'
+import SustainabilitySVG from '@/assets-webkit/img/landing/subjects-sustainability.svg'
+import { Link } from '@/components/content/link'
+import { LandingSubjectLink, LandingSubjectsData } from '@/data-types'
+import { makeResponsivePadding, makeTransparentButton } from '@/helper/css'
+
+interface LandingSubjectsProps {
+  data: LandingSubjectsData
+}
+
+export function LandingSubjectsNew({ data }: LandingSubjectsProps) {
```

Comment:
This is almost identical to the current one (we still use on e.g. es.serlo.org). 
Probably one of the first files where we can get rid of the code duplication if we move forward with this.